### PR TITLE
add-time-to-market-insights content

### PR DESCRIPTION
**Pull request by alan-akeneo-2020**

### CODE REVIEW

File: content/md/productivity/time-to-market-insights.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+4. The control panel to define the metric and the associated filters you want to analyze
+
+
+## Control panel
+
+![TTMI control panel](../img/TTMI_control_panel.png)
+
+### Metrics
+
+2 metrics to analyze your time-to-market performance
```
Comment (felix-couprie) [inline content/md/productivity/time-to-market-insights.md:39]:
```suggestion
2 metrics to analyze your time-to-market performance:
```

File: content/md/productivity/time-to-market-insights.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- Number of enriched products: total number of products enriched over the defined period of time
+
+### Dimensions
+
+You can analyze the selected metric by grouping through entities: Families and Categories.
+- Family: how you have defined completeness for products according to a set of required Attributes to be enriched
+- Category: how you have organized and classified your products in Akeneo PIM
+
+Also, you can go deeper into your analysis through the channels and the locales: 
+- All: select “All Channels” or “All Locales” to get a broad overview of your product catalog across the board
```
Comment (felix-couprie) [inline content/md/productivity/time-to-market-insights.md:50]:
```suggestion
- All: select `All Channels` or `All Locales` to get a broad overview of your product catalog across the board
```

File: content/md/productivity/time-to-market-insights.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+- At Least one: choose “At least one Channel” or “At least one Locale”
```
Comment (felix-couprie) [inline content/md/productivity/time-to-market-insights.md:51]:
```suggestion
- At Least one: choose `At least one Channel` or `At least one Locale` 
```

File: content/md/productivity/time-to-market-insights.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+## Dashboard and table vizualisation
+
+![TTMI graph and table](../img/TTMI_graph_table.png)
+
+Let’s analyze the capabilities available through the dashboard and the table visualization
+
+### Dashboard
+
+The **legend** summarizes the data defined in the configuration panel and is displayed in the graph below
```
Comment (felix-couprie) [inline content/md/productivity/time-to-market-insights.md:66]:
```suggestion
The **caption** summarizes the data defined in the configuration panel and is displayed in the graph below
```

File: content/md/productivity/time-to-market-insights.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+![TTMI legend](../img/TTMI_legend.png)
+
+The **pop-up** available once hovering on the chart displays details about the data you have chosen
+![TTMI popup](../img/TTMI_graph_pop_up.png)
+
+
+### Table
+
+The **search bar** allows finding a specific Family. For the Category, it should be available soon.
```
Comment (felix-couprie) [inline content/md/productivity/time-to-market-insights.md:75]:
```suggestion
The **search bar** allows finding a specific family. Categories will be available soon.
```

File: content/md/productivity/time-to-market-insights.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+![TTMI search](../img/TTMI_search.png)
+The Family or Category lines are clickable and allow for updating the chart above with your chosen data set
```
Comment (felix-couprie) [inline content/md/productivity/time-to-market-insights.md:77]:
```suggestion
The family and the category rows are clickable and allow updating the chart above with your chosen data set.
```

File: content/md/productivity/time-to-market-insights.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+::: info
+All the data prior to 15th December 2022 is not available as we have started to track the usage at this period. This condition is the same for the product created before this period.
```
Comment (felix-couprie) [inline content/md/productivity/time-to-market-insights.md:80]:
```suggestion
All the data before 15th December 2022 is unavailable as we have started tracking the usage at this period. This condition is the same for all products created before this period. 
```

File: content/md/productivity/time-to-market-insights.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
+---
+id: time-to-market-insights
+themes: boost-your-productivity
+title: Time-to-market insights
+ee: true
+ge: false
+related:
```
Comment (felix-couprie) [inline content/md/productivity/time-to-market-insights.md:7]: I suppose it's normal not to have any related topic?

File: content/md/productivity/time-to-market-insights.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+- Category: how you have organized and classified your products in Akeneo PIM
+
+Also, you can go deeper into your analysis through the channels and the locales: 
+- All: select “All Channels” or “All Locales” to get a broad overview of your product catalog across the board
+- At Least one: choose “At least one Channel” or “At least one Locale” 
+- Choose a specific channel or locale for focused insights in a particular area
+
+### Time frame (From)
+
+You can define the time frame you want to display for your analysis
```
Comment (jnash-akeneo) [inline content/md/productivity/time-to-market-insights.md:56]:
```suggestion
You can define the time frame you want to display for your analysis.
```

File: content/md/productivity/time-to-market-insights.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+## Dashboard and table vizualisation
+
+![TTMI graph and table](../img/TTMI_graph_table.png)
+
+Let’s analyze the capabilities available through the dashboard and the table visualization
```
Comment (jnash-akeneo) [inline content/md/productivity/time-to-market-insights.md:62]:
```suggestion
Let’s analyze the capabilities available through the dashboard and the table visualization.
```

File: content/md/productivity/time-to-market-insights.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+
+### Dashboard
+
+The **legend** summarizes the data defined in the configuration panel and is displayed in the graph below
+![TTMI legend](../img/TTMI_legend.png)
+
+The **pop-up** available once hovering on the chart displays details about the data you have chosen
```
Comment (jnash-akeneo) [inline content/md/productivity/time-to-market-insights.md:69]:
```suggestion
Hover over the chart to display a **pop-up** that displays details about the data you have selected.
```

File: content/md/productivity/time-to-market-insights.md (hunk 8 above)
Comment (alan-akeneo-2020) [reply content/md/productivity/time-to-market-insights.md:7]: Yes, as a new topic, I did not identify any relevant topic to be related to TTMI :)

**Review by felix-couprie — APPROVED** [review]
GTM

**Pull request merged**